### ISSUE TITLE
Flaky e2e test: create-first-project-revision

### ISSUE THREAD

**Issue by dleard** [bcgov/cas-cif#1375]
We have a flaky e2e test that sometimes passes and sometimes fails. It looks like sometimes it is detecting an accessibility violation in the `cif/project-revision/create-first-project-revision.spec.js` spec. It is also timing out on finding an element at the same time.

The likely issue is that it is catching a page transition during the cypress-axe check and then getting stuck so that it can't find the element it's looking for.
